### PR TITLE
Add unit tests for azure_client.py

### PR DESCRIPTION
**Pull request by acltang**
image

Ignoring `upload_file_to_azure` until it is fixed.

### CODE REVIEW

File: azure_client/azure_client.py [hunk 1]
```diff
@@ -40,16 +40,17 @@ def create_uuid_container_in_azure(uuid):
...
-    audios_container = uuid + "-audios"
-    files_container = uuid + "-files"
-    videos_container = uuid + "-videos"
-
-    block_blob_service.create_container(images_container)
-    block_blob_service.create_container(audios_container)
-    block_blob_service.create_container(files_container)
-    block_blob_service.create_container(videos_container)
-    logger.debug("user folder creation successful")
+    if utility.verify_uuid(uuid):
```
Comment (acltang) [inline azure_client/azure_client.py]: The UUID is already checked in `CreateUserFolder`, but is checked here again for unit testing.

File: logger/test_logger.py [hunk 2]
```diff
@@ -13,6 +13,8 @@
                          ]
                          )
 def test_request_service(capsys, input, expected_output):
+    logger.log_stream.truncate(0)
+    logger.log_stream.seek(0)
```
Comment (acltang) [inline logger/test_logger.py]: The inclusion of `test_azure_client.py` breaks the logger unit test due to the order in which the unit tests are run. `test_azure_client.py` writes to `stderr` before `test_logger.py` is run, causing this unit test to read garbage from `stderr`. These 2 lines clear `stderr` before running the test.

File: azure_client/test_azure_client.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+from azure_client import azure_client
+import random
+import string
+
+valid_uuid_1 = "".join(random.choices(string.ascii_lowercase + string.digits, k=26))
+
+
+@pytest.mark.parametrize("input_uuid, expected_output",
+                         [
+                             # testing with valid uuid
```
Comment (faraonc) [inline azure_client/test_azure_client.py]: Add test `"desc"`

File: azure_client/test_azure_client.py [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+                             (valid_uuid_1, True),
+                             # testing with duplicate uuid
+                             (valid_uuid_1, True),
+                             # testing with invalid uuid
+                             ("1234abcd5454efef8842llaaaab", False),
```
Comment (faraonc) [inline azure_client/test_azure_client.py]: Make a global `invalid_uuid`

**Pull request merged**